src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -59,7 +55,28 @@ public class CoralArm extends SubsystemBase {
   private double currentAngleTime = 0;
 
   /** Creates a new Arm. */
-  public CoralArm() {
+  public Arm(ArmParameters PARAMETERS) {
+
+    MASS = PARAMETERS.getMass();
+    GEAR_RATIO = PARAMETERS.getGearRatio();
+    MOTOR = PARAMETERS.getMotorParameters();
+
+    EFFICIENCY = PARAMETERS.getEfficiency();
+    RADIANS_PER_REVOLUTION = PARAMETERS.getRadiansPerRevolution();
+    MAX_ANGULAR_SPEED = EFFICIENCY * MOTOR.getFreeSpeedRPM() * RADIANS_PER_REVOLUTION / 60.0;
```

Comment:
Since the maximum angular speed and acceleration are calculated from the arm parameters, we could move the calculations to the `ArmParameters` enum. Also, the speed calculation is wrong. (My bad!)

```java
public double getMaxAngularSpeed() {
  return (this.efficiency * this.motor.freeSpeedRadPerSec) / this.gearRatio;
}

public double getMaxAngularAcceleration() {
  return (this.efficiency * 2 * this.motor.stallTorqueNewtonMeters * this.gearRatio) / (this.mass * this.armLength);
}
```